pydis_site/static/js/timeline/main.js

```diff
@@ -32,17 +32,36 @@
 		var self = this;
 		for( var i = 0; i < this.blocks.length; i++) {
 			(function(i){
-				if( self.contents[i].classList.contains("cd-timeline__content--hidden") && self.blocks[i].getBoundingClientRect().top <= window.innerHeight*self.offset ) {
+				if((self.contents[i].classList.contains("cd-timeline__content--hidden") || self.contents[i].classList.contains("cd-timeline__content--bounce-out")) && self.blocks[i].getBoundingClientRect().top <= window.innerHeight*self.offset ) {
 					// add bounce-in animation
 					self.images[i].classList.add("cd-timeline__img--bounce-in");
 					self.contents[i].classList.add("cd-timeline__content--bounce-in");
 					self.images[i].classList.remove("cd-timeline__img--hidden");
 					self.contents[i].classList.remove("cd-timeline__content--hidden");
+					self.images[i].classList.remove("cd-timeline__img--bounce-out");
+					self.contents[i].classList.remove("cd-timeline__content--bounce-out");
 				}
 			})(i);
 		}
 	};
 
+	VerticalTimeline.prototype.hideBlocksScroll = function () {
+		if ( ! "classList" in document.documentElement ) {
+			return;
+		}
+		var self = this;
+		for( var i = 0; i < this.blocks.length; i++) {
+			(function(i){
```

Comment:
I'd probably nitpick a little on the weird floaty var and exclamation mark - but I guess this is the prevailing style in this file, so I suppose it's fine.

Reply:
I copy-pasted this part.